aea/helpers/preference_representations/base.py

```diff
@@ -22,26 +22,20 @@
 import math
 from typing import Dict
 
-from aea.exceptions import enforce
```

Comment:
By making the argument an internal variable, its value could still be changed by now it would be consistently applied in all function calls. 

The downside is that this function is now specific to TAC and not a generic `logarithmic_utility` function (which might originally have been intended like this anyway in which case this is a non-issue)

Reply:
You can maintain the argument and set its default value to 100. Although still overridable it is less likely misused as you removed it in gop.